sql/backfill.go

```diff
@@ -373,12 +406,18 @@ func (sc *SchemaChanger) truncateAndBackfillColumnsChunk(
 	return curIndexKey.PrefixEnd(), done, err
 }
 
+// IndexTruncateChunkSize is the maximum number of rows processed per chunk
+// during an index truncation.
+const IndexTruncateChunkSize = 1000
```

Comment:
Do you see value in tuning all these chunk sizes separately? The range between them isn't that large so it seems like it would be simpler to reduce this to one value (or maybe two, for deletions vs additions). In any case, put all of these constants together at the top of the file so we can see how big they all are in relation to each other.